### ISSUE TITLE
[bytes counter] when use EwmaSpeed ,speed and remain time calculate is wrong?

### ISSUE THREAD

**Issue by lixd** [vbauerster/mpb#120]
the bar like this:

```text
1.89 GiB / 2.00 GiB 95 % [======================================================>---]  ] 0s8.06 GiB/s
```

time is 0s,speed is  8.06 GiB/s，but really speed is less than 600MiB/s

when use AverageSpeed,is ok,bar like this:
```text
1.94 GiB / 2.00 GiB 97 % [=======================================================>--]  ] 548.14 MiB/s
```
and test [ bytes counter example](https://github.com/vbauerster/mpb/blob/master/_examples/io/main.go) ,is ok.

my code like this
```go
        // create a tmp file to read
	file, err := os.Create("./tmp.txt")
	if err != nil {
		return
	}
	data := make([]byte, 1024*1024*1024*2)
	file.Write(data)
	file.Close()
	defer func() {
		os.Remove("./tmp.txt")
		os.Remove("./tmp2.txt")
	}()

	p := mpb.New(
		mpb.WithWidth(60),
		mpb.WithRefreshRate(180*time.Millisecond),
	)

	bar := p.New(0,
		mpb.BarStyle(),
		mpb.PrependDecorators(
			decor.CountersKibiByte("% .2f / % .2f"),
			decor.Percentage(decor.WCSyncSpace),
		),
		mpb.AppendDecorators(
			decor.Name(" ] "),
			decor.EwmaETA(decor.ET_STYLE_GO, 90),
			decor.EwmaSpeed(decor.UnitKiB, "% .2f", decor.DSyncSpace),
			//decor.AverageSpeed(decor.UnitKiB, "% .2f", decor.WCSyncWidth),
		),
	)

	// file to reader
	open, err := os.Open("./tmp.txt")
	if err != nil {
		panic(err)
	}
	stat, err := open.Stat()
	if err != nil {
		panic(err)
	}
	// set total as file size
	fmt.Println("size: ", stat.Size())
	bar.SetTotal(stat.Size(), false)

	// create proxy reader
	proxyReader := bar.ProxyReader(open)
	//defer proxyReader.Close()


	// copy from proxyReader, ignoring errors
	create, err := os.Create("./tmp2.txt")
	if err != nil {
		panic(err)
	}
	defer create.Close()

	n, _ := io.Copy(create, proxyReader)


	// triggering complete event now
	bar.SetTotal(-1, true)
```

**Comment by vbauerster** [comment]
Please refer to documentation of [ewma](https://github.com/VividCortex/ewma) lib.

**TLDR**
You have to pick correct weight parameter. If you choose anything than 30, ewma lib will collect 10 samples before any result. In your code sample you've chosen 90 and got 0 result, which means there were less than 10 samples.

**Comment by lixd** [comment]
@vbauerster thanks for your replay,but why this example[ bytes counter example](https://github.com/vbauerster/mpb/blob/master/_examples/io/main.go)  it ok, params in my code is same with this. what need i do to fix my code? help~

**Comment by vbauerster** [comment]
use 0 as arg to ewma, that will default to implementation that doesn't require prerequisite samples.

**Comment by lixd** [comment]
Thanks,bug u nfortunately，i tried use 0 as arg to ewma,,but not work too.
